packages/helpers/qrcode-modal/src/browser/components/Modal.tsx

```diff
@@ -32,12 +33,16 @@ function Modal(props: ModalProps) {
           </div>
         </div>
         <div>
-          {displayQRCode ? <QRCodeDisplay uri={props.uri} /> : <DeepLinkDisplay uri={props.uri} />}
+          {displayQRCode ? (
```

Comment:
Not major but if you wanted to keep it on a single line and DRY you could declare at the top
```js
const { text, uri } = props;
const displayProps = { text, uri }
```

This would then be
```suggestion
          {displayQRCode ? <QRCodeDisplay { ...displayProps } /> : <DeepLinkDisplay { ...displayProps } />}
```

Reply:
Actually that’s a good idea

Reply:
Added it!